logstash/logstash_entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Ensure FAKTORY_CRON_TOML_BASE64 is set, if not, we shouldn't start the service.
```

Comment:
Comment may be out of date as it references FAKTORY

Reply:
doh